lib/activerecord-import/import.rb

```diff
@@ -941,7 +941,7 @@ def load_association_ids(model)
           association = association.target
           next if association.blank? || model.public_send(column_name).present?
 
-          association_primary_key = Array(association_reflection.association_primary_key)[column_index]
+          association_primary_key = Array(association_reflection.association_primary_key.tr("[]:", "").split(", "))[column_index]
```

Comment:
Without a composite PK, the association_reflection.association_primary_key has a value of the format "id", whereas with a composite PK in ActiveRecord 7.1 the value is of the format "[:id, :account_id]" (again a String). The .tr loses the [, :, ] characters, so that the .split(", ") will give us an array of the two colums: ["id", "account_id"]